pkgs/applications/video/mpv/default.nix

```diff
@@ -81,59 +72,28 @@
   cmsSupport ? true,
   drmSupport ? stdenv.hostPlatform.isLinux,
   dvbinSupport ? stdenv.hostPlatform.isLinux,
-  dvdnavSupport ? stdenv.hostPlatform.isLinux,
+  dvdnavSupport ? true,
   jackaudioSupport ? false,
   javascriptSupport ? true,
-  libpngSupport ? true,
   openalSupport ? true,
-  pipewireSupport ? stdenv.hostPlatform.isLinux,
-  pulseSupport ? config.pulseaudio or stdenv.hostPlatform.isLinux,
+  pipewireSupport ? !stdenv.hostPlatform.isDarwin,
+  pulseSupport ? config.pulseaudio or (!stdenv.hostPlatform.isDarwin),
```

Comment:
Why is `config.pulseaudio` still employed?

Reply:
I think this change shouldn’t really affect the semantics there, right? I assume it is so that PulseAudio haters can set `config.pulseaudio = false;` globally.

Reply:
Afaicr this "global enablement" was abandoned decades ago, but the cargo cult was kept around pulseaudio.
I never understood why.

But since it can modify semantics, let's remove it another time.

Reply:
Yeah I’m not attached to it myself – personally I’d rather drop the vast majority of these flags entirely, I hate combinatorial explosions of build conditionals – but I focused just on avoiding Linux bias and dropping flags that were somewhere between broken and useless for now.

Reply:
I agree with you both. Let's just keep it for now. Using the flag probably breaks stuff in other places anyway. :sweat_smile: